cloud_Azure/terraform/module/examples/multiple_accounts_multiple_resource_group/azure_onboarder.py

```diff
@@ -49,20 +49,16 @@ def execute_action(action: TerraformAction, profiles: List[AzureProfile]) -> boo
 
 # az login is required prior to calling terraform: "get_nsg.py" uses Azure CLI to gather Network Security Group names
 def azure_login(profile: AzureProfile) -> bool:
-    azure_login_command = f"login --service-principal -u {profile.principal_id} -p {profile.principal_secret} --tenant {profile.tenant_id}"
-    return_code, _, logs = az(azure_login_command)
-    if return_code != EX_OK:
-        log.warning(
-            "Failed to sign into Azure using profile '%s' credentials. Skipping profile. Error message: '%s'",
-            profile.name,
-            logs,
-        )
+    command = f"login --service-principal -u {profile.principal_id} -p {profile.principal_secret} --tenant {profile.tenant_id}"  # returns a list
```

Comment:
The comment should be on the next line.

Reply:
the comment relates to the command specifically